qa/repository-multi-version/src/test/java/org/opensearch/upgrades/MultiVersionRepositoryAccessIT.java

```diff
@@ -141,14 +143,14 @@ public void testCreateAndRestoreSnapshot() throws IOException {
                 case STEP2_NEW_CLUSTER:
                 case STEP4_NEW_CLUSTER:
                     assertSnapshotStatusSuccessful(client, repoName,
-                        snapshots.stream().map(sn -> (String) sn.get("snapshot")).toArray(String[]::new));
+                        snapshots.stream().map(sn -> (String) sn.get("snapshot")).toArray(String[]::new), Strings.EMPTY_ARRAY);
                     break;
                 case STEP1_OLD_CLUSTER:
-                    assertSnapshotStatusSuccessful(client, repoName, "snapshot-" + TEST_STEP);
+                    assertSnapshotStatusSuccessful(client, repoName,  new String[] {"snapshot-" + TEST_STEP}, Strings.EMPTY_ARRAY);
```

Comment:
In this test, what does old cluster refer to? Will old cluster understand indices list?

Reply:
In the IT we are passing an empty array for indices which which is only to account for the change in the constructor of SnapshotsStatusRequest. It is internally not being consumed by the request [converter](https://github.com/opensearch-project/OpenSearch/blob/main/client/rest-high-level/src/main/java/org/opensearch/client/SnapshotRequestConverters.java#L178-L182) for the snapshots rest client. This should ensure no breakage in the tests.